examples/00_misc/05_standalone_field.py

```diff
@@ -0,0 +1,27 @@
+"""
+Standalone Field class
+----------------------
+
+The :any:`Field` class of GSTools can be used to plot arbitrary data in nD.
+
+In the following example we will produce 10000 random points in 4D with
+random values and plot them.
+"""
+import numpy as np
+import gstools as gs
+
+x1 = np.random.RandomState(19970221).rand(10000) * 100.0
+x2 = np.random.RandomState(20011012).rand(10000) * 100.0
+x3 = np.random.RandomState(20210530).rand(10000) * 100.0
+x4 = np.random.RandomState(20210531).rand(10000) * 100.0
+values = np.random.RandomState(2021).rand(10000) * 100.0
```

Comment:
I think it would be nice to utilise our MasterRNG here to generate the different RNG-streams, instead of instantiating 5 single ones on your own. On the other hand, this clutters the example a bit. What do you think?

Reply:
Why not. What do you think about this:
```python
rng = gs.random.RNG(19970221)
x1 = rng.random.rand(10000) * 100.0
x2 = rng.random.rand(10000) * 100.0
x3 = rng.random.rand(10000) * 100.0
x4 = rng.random.rand(10000) * 100.0
values = rng.random.rand(10000) * 100.0
```
?

Reply:
That looks a lot nicer!

Reply:
Hmm, now that I thought about it again, a single RNG stream for all the values shouldn't be a problem at all. It's only a problem, if you take parts of the generated numbers and separate them, like
```python
for t in time:
    x += rng()
    y += rng()
````
There you should be using
```python
for t in time:
    x += rng1()
    y += rng2()
````
But in your example you take all the subsequent generated numbers for one dimension and then for the next.
So, you could use

```python
rng = np.random.RandomState(19970221)
x1 = rng.random.rand(10000) * 100.0
x2 = rng.random.rand(10000) * 100.0
x3 = rng.random.rand(10000) * 100.0
x4 = rng.random.rand(10000) * 100.0
values = rng.random.rand(10000) * 100.0
```

Reply:
Will use the latter one. with one instance of `RandomState`.